src/app/components/article/MediaArticleCard.js

```diff
@@ -122,9 +123,9 @@ MediaArticleCard.propTypes = {
   id: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   url: PropTypes.string,
-  date: PropTypes.number.isRequired, // Timestamp
+  date: PropTypes.instanceOf(Date).isRequired,
```

Comment:
Question: These values are passed as `article.updated_at` which is a timestamp string... does this conversation happen somewhere else?

Reply:
AMOQ: Ah, I see you fixed them in `<MediaArticlesDisplay />` :+1: